app/screens/channel/index.tsx

```diff
@@ -1,28 +1,46 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
 
-import React from 'react';
-import {SafeAreaView, ScrollView, StatusBar, StyleSheet, Text, View} from 'react-native';
-import {
-    Colors,
-    DebugInstructions,
-    Header,
-    LearnMoreLinks,
-    ReloadInstructions,
-} from 'react-native/Libraries/NewAppScreen';
-
-import type {LaunchProps} from '@typings/launch';
-
-import {logout} from '@actions/remote/general';
```

Comment:
I know this does not belong here, but perhaps can we leave a way to logout until the sidebar is included.

Reply:
I have added it.  We should also be resetting the navigation to 'SelectServer' screen. Correct? What kind of launchProps should we give it ?

Reply:
no no, that is handled automatically when you logout ;)

Reply:
I notice a DB crash on logging out; saying channel XXX does not exist. It might be that WDB tries to re-render because the DB changed.  We can look into how to gracefully do it in the coming PRs.